docs/powerauth-server/PowerAuth-Server-1.3.0.md

```diff
@@ -260,3 +260,18 @@ We consider the 5-minute interval to still be safe, since the relatively high ac
 ```
 powerauth.service.crypto.activationValidityInMilliseconds=120000
 ```
+
+## Database Dialect Configuration
+
+The latest release of PowerAuth requires configuration of database dialect.
+
+The dialect is specified using following configuration property:
+```properties
+spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect
```

Comment:
We are using a deprecated value in the example. Could we change it?

Reply:
Sure, good idea.